core/src/main/scala/org/broadinstitute/dsde/rawls/serviceperimeter/ServicePerimeterService.scala

```diff
@@ -30,6 +30,17 @@ class ServicePerimeterService(dataSource: SlickDataSource, gcsDAO: GoogleService
     dataAccess.workspaceQuery.getWorkspacesInPerimeter(servicePerimeterName)
   }
 
+  /**
+    * In its own transaction, look up all of the Billing Projects that use the specified
```

Comment:
I don't think this is in its own transaction, right?  The `dataAccess` being passed in is someone else's transaction, right?

Reply:
great point! we copied this from the workspace query, and I think you're right about that so I updated both.